apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -1,7 +1,16 @@
 import React, {Component} from 'react';
+//import PropTypes from 'prop-types';
```

Comment:
nit: remove this?

Reply:
It's going to be re-added when I start including real comments as props, which is imminently 🤞